lib/features/home/home.dart

```diff
@@ -152,22 +152,17 @@ class _HomePageState extends State<HomePage> with TrayListener, WindowListener {
   ///window manager methods
   void _initWindowManager() async {
     windowManager.addListener(this);
-    await windowManager.setPreventClose(true);
-    WindowOptions windowOptions = const WindowOptions(
-      size: ui.Size(360, 712),
-      minimumSize: ui.Size(360, 712),
-      maximumSize: ui.Size(360, 712),
-      center: true,
-      backgroundColor: Colors.transparent,
-      skipTaskbar: false,
-      windowButtonVisibility: true,
-    );
-    windowManager.waitUntilReadyToShow(windowOptions, () async {
-      await windowManager.show();
-      await windowManager.focus();
-      await windowManager.setResizable(false);
+    if (Theme.of(context).platform != TargetPlatform.windows) return;
+    // temporary workaround for an issue on Windows where the UI loads with a
+    // distorted view on start. The problem goes away when the window is
+    // resized. See https://github.com/leanflutter/window_manager/issues/464
+    await Future<void>.delayed(const Duration(milliseconds: 100), () {
+      windowManager.getSize().then((ui.Size value) {
+        windowManager.setSize(
+          ui.Size(value.width + 1, value.height + 1),
+        );
+      });
```

Comment:
Here's the primary fix for the broken UI on Windows which is a temporary workaround until a new version of window_manager is released.